whitepapers/game-theory.md

```diff
@@ -0,0 +1,47 @@
+# Olympus Game Theory
+
+kleb
```

Comment:
What is this?

Reply:
An essay analysing the game theoretic elements of Olympus. You can view and comment on the draft from the whitepaper channel on discord. It's very much a WIP. First few sections are ready to be uploaded.

Reply:
My question was why it states `kleb` there.